platformio.ini

```diff
@@ -11,7 +11,9 @@
 [env]
 upload_speed = 921600
 monitor_speed = 115200 ; must remain at 115200 for improv
+;monitor_speed = 74880  ; hardware default
 board_build.filesystem = littlefs
+board_build.f_flash = 26000000L
```

Comment:
I don't want to merge this unless we find it absolutely necessary.

Reply:
I actually think that this is the most important change to get wide testing.  Reducing from 40Mhz to 26Mhz could be the key to avoiding flash CRC errors.  Both my systems crashed this week unrelated to firmware upload.  Just maybe slowing down the flash chip frequency could make things more reliable.